src/libs/PolicyUtils.ts

```diff
@@ -1080,6 +1081,13 @@ function goBackWhenEnableFeature(policyID: string) {
     }, CONST.WORKSPACE_ENABLE_FEATURE_REDIRECT_DELAY);
 }
 
+function navigateToExpensifyCardPage(policyID: string) {
+    setTimeout(() => {
+        Navigation.navigate(ROUTES.WORKSPACE_EXPENSIFY_CARD.getRoute(policyID));
```

Comment:
```suggestion
Navigation.setNavigationActionToMicrotaskQueue(() => Navigation.navigate(ROUTES.WORKSPACE_EXPENSIFY_CARD.getRoute(policyID)));
        
```

If this is added to prevent the brief “not found” page because the navigation action might have occurred before the Onyx update enabling the Expensify Cards page, we can add the navigation task to the microtask queue to ensure it happens only after the Onyx update using `setNavigationActionToMicrotaskQueue` and avoid using `setTimeout`.